prefetch.bs

```diff
@@ -129,48 +130,73 @@ A <dfn>redirect chain</dfn> is a [=list=] of [=exchange records=].
 
 <hr>
 
-Each {{Document}} has a <dfn export>prefetch buffer</dfn>, which is a [=list=] of [=prefetch records=].
+Each {{Document}} has <dfn export for="Document">prefetch records</dfn>, which is a [=list=] of [=prefetch records=].
 
-A <dfn>prefetch record</dfn> is a [=struct=] with the following [=struct/items=]:
+A <dfn export>prefetch record</dfn> is a [=struct=] with the following [=struct/items=]:
 * <dfn export for="prefetch record">URL</dfn>, a [=URL=]
 * <dfn export for="prefetch record">referrer policy</dfn>, a [=referrer policy=]
-* <dfn export for="prefetch record">sandbox flags present</dfn>, a boolean
-* <dfn export for="prefetch record">redirect chain</dfn>, a [=redirect chain=]
-* <dfn export for="prefetch record">expiry time</dfn>, a {{DOMHighResTimeStamp}}
+* <dfn export for="prefetch record">anonymization policy</dfn>, a [=prefetch IP anonymization policy=]
+* <dfn export for="prefetch record">initiator</dfn>, a {{DOMString}}
```

Comment:
If this is not a https://fetch.spec.whatwg.org/#concept-request-initiator then it is a bad name.

Reply:
It serves a similar purpose, and even Fetch already has two:

https://fetch.spec.whatwg.org/#concept-request-initiator
https://fetch.spec.whatwg.org/#request-initiator-type

I've renamed it "label" (every conceivable synonym is used somewhere in HTML or a related spec), but open to other names.